src/components/LogIn/LogIn.test.tsx

```diff
@@ -58,38 +31,6 @@ describe("LogIn", () => {
     });
     renderComponent(<LogIn>App content</LogIn>, { state });
     expect(document.querySelector(".login")).not.toBeInTheDocument();
-  });
-
-  it("renders an IdentityProvider login UI if the user is not logged in", () => {
-    const state = rootStateFactory.build({
-      general: generalStateFactory.withConfig().build({
-        visitURLs: ["I am a url"],
-        config: configFactory.build({
-          identityProviderAvailable: true,
-        }),
-      }),
-    });
-    renderComponent(<LogIn>App content</LogIn>, { state });
-    expect(
-      screen.getByRole("link", { name: "Log in to the dashboard" }),
-    ).toBeInTheDocument();
-    const content = screen.getByText("App content");
-    expect(content).toBeInTheDocument();
-    expect(content).toHaveClass("app-content");
-  });
-
-  it("renders a UserPass login UI if the user is not logged in", () => {
-    const state = rootStateFactory.build({
-      general: generalStateFactory.withConfig().build({
-        config: configFactory.build({
-          identityProviderAvailable: false,
-        }),
-      }),
-    });
-    renderComponent(<LogIn>App content</LogIn>, { state });
-    expect(screen.getByRole("button")).toHaveTextContent(
-      "Log in to the dashboard",
-    );
```

Comment:
We still need tests in this file about which form is displayed as the logic for choosing the form based on the state of `identityProviderAvailable` is in the Login component.